lib/http_fuzzer/engine.py

```diff
@@ -338,7 +350,8 @@ def sample_event_key_evaluator(response, payload, value):
     """
     try:
         if value != '':
-            exec ("value = " + value)
+            exec ("global value1; value1 = " + value)
```

Comment:
value1 needs to initialised - otherwise flake8 and pylint report error:
```
************* Module lib.http_fuzzer.engine
lib/http_fuzzer/engine.py:354:20: E0602: Undefined variable 'value1' (undefined-variable)
```